sdks/fern/definition/actor/builds.yml

```diff
@@ -100,6 +100,7 @@ types:
   PrepareBuildRequest:
     properties:
       name: string
+      tags: unknown
```

Comment:
we can't pass tags in prepare bc it'd mark builds as `current` before the build is complete

Reply:
What do you propose?

Reply:
leave it as is. make the client call patch tags after uploaded. we can do something cleaner later.